Calibration/LumiAlCaRecoProducers/plugins/AlcaPCCEventProducer.cc

```diff
@@ -37,7 +37,8 @@ class AlcaPCCEventProducer : public edm::global::EDProducer<> {
 
 private:
   const edm::InputTag pixelClusterLabel_;
-  const std::string trigstring_;  //specifies the trigger Rand or ZeroBias
+  const std::string trigstring_;      //specifies the trigger Rand or ZeroBias
+  const bool savePerROCInfo_ = true;  // save per ROC data (important for the special fills)
```

Comment:
Now you no longer need ```= true``` as it gets set in the class constructor (like the other const parameters).

Reply:
What a shame, of course